src/routes/settings/SyncNostrContacts.tsx

```diff
@@ -105,12 +104,7 @@ export function SyncNostrContacts() {
         setError(undefined);
         setLoading(true);
         try {
-            if (!PRIMAL_API) throw new Error("PRIMAL_API not set");
```

Comment:
says you need to remove the `PRIMAL_API` definition on this page

Reply:
good catch